apps/designer/app/canvas/canvas.tsx

```diff
@@ -182,7 +186,9 @@ export const Canvas = ({ data }: CanvasProps): JSX.Element | null => {
   const isPreviewMode = useSubscribePreviewMode();
   const elements = useElementsTree();
 
-  if (elements === undefined) return null;
```

Comment:
did the rule make it completely impossible to make single line if?

Reply:
Several [options](https://eslint.org/docs/latest/rules/curly) are available to force it only in some instances. 
Forcing for every case is reasonable to have consistency.